tests/e2e/test_e2e_template.py

```diff
@@ -607,7 +607,7 @@ def test_make_hypertrain(
     n = 2
     with finalize("make kill-hypertrain-all"):
         out = run(
-            f"make hypertrain N_HYPERPARAM_JOBS={n}",
+            f"make hypertrain N_HYPERPARAM_JOBS={n} TRAIN_CMD='sleep 10'",
```

Comment:
this test flakes sometimes: for exmaple https://dev.azure.com/neuromation/cookiecutter-neuro-project/_build/results?buildId=3065